bridgesync/migrations/bridgesync0005.sql

```diff
@@ -0,0 +1,8 @@
+-- +migrate Down
+ALTER TABLE bridge DROP COLUMN tx_sender;
+ALTER TABLE claim DROP COLUMN tx_sender;
+
+-- +migrate Up
+ALTER TABLE bridge ADD COLUMN tx_sender VARCHAR;
+ALTER TABLE claim ADD COLUMN tx_sender VARCHAR;
```

Comment:
Should we make it non-nullable?

```suggestion
ALTER TABLE bridge ADD COLUMN tx_sender VARCHAR NOT NULL;
ALTER TABLE claim ADD COLUMN tx_sender VARCHAR NOT NULL;
```